Data interface chris

CLucas96 opened the pull request:
Added ListBox to show Raw data from the Fact Table along with tidying up the First Tab.

Commit:
Added Labels identifying buttons, Named the tabs and Form.

Commit:
Added a ListBox to show the Raw Data from the Fact Table. ListBoxFactTable Loads after clikcing the "Populate Fact Table" Button.

MattyJP commented:
Impressed with presentation of data population tab. Few issues to address:
- listBoxFactTable's time to be formatted
- listBoxFactTable to not be populated until all three dimensions created.

Will push a commit to alter these.

MattyJP committed:
Current Changes:
- Removed blank event handler for label2_click.
-  Created CheckDimensionsPopulated function which enables the fact table creation button after the other dimension buttons are disabled (after they are used).
- Created RemoveBlankTime function to remove repeated code and out of ill-fated attempt to remove blank time value from listBoxFactTable. Function works fine in other contexts, it is just due to the nature of the fact table's list creation.

CLucas96 commented:
Commit Changes work as intended. 
Time removal from the fact table seemed troublesome when i tried to resolve issue. 
system is stable and is ready to merge
